openbas-model/src/main/java/io/openbas/helper/InjectModelHelper.java

```diff
@@ -50,46 +50,58 @@ public static boolean isReady(
     }
 
     ObjectNode contractContent = injectorContract.getConvertedContent();
-    List<JsonNode> contractMandatoryFields =
+    List<JsonNode> contractFields =
         StreamSupport.stream(contractContent.get(CONTACT_CONTENT_FIELDS).spliterator(), false)
-            .filter(
-                field -> {
-                  String key = field.get(CONTACT_ELEMENT_CONTENT_KEY).asText();
-                  boolean isMandatory = field.get(CONTACT_ELEMENT_CONTENT_MANDATORY).asBoolean();
-                  boolean isMandatoryGroup =
-                      field.hasNonNull(CONTACT_ELEMENT_CONTENT_MANDATORY_GROUPS)
-                          && field.get(CONTACT_ELEMENT_CONTENT_MANDATORY_GROUPS).asBoolean();
-                  return key.equals(CONTACT_ELEMENT_CONTENT_KEY_ASSETS)
-                      || isMandatory
-                      || isMandatoryGroup;
-                })
             .toList();
 
     boolean isReady = true;
-    for (JsonNode jsonField : contractMandatoryFields) {
+    for (JsonNode jsonField : contractFields) {
```

Comment:
What do we do for mandatory teams ?
I am not sure to understand the new implementation.